src/components/StorylaneDemo/DemoComponent.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+interface DemoProps {
+    url: string;
+    title: string;
+  }
+
+export default function DemoComponent({ url, title }: DemoProps) {
+  return (
+    <div>
+      <script src="https://js.storylane.io/js/v1/storylane.js"></script>
```

Comment:
```suggestion
      <script asyc src="https://js.storylane.io/js/v1/storylane.js"></script>
```
Probably best we add async so the other page content is not blocked